src/it/junit5-native/build.gradle

```diff
@@ -0,0 +1,27 @@
+plugins {
+    id 'java'
+    id 'io.qameta.allure'
+}
+
+repositories {
+    mavenCentral()
+    jcenter()
+}
+
+allure {
+    version = '2.4.1'
+    aspectjweaver = true
+
+    useJUnit5 {
+        version = "2.0-BETA21"
```

Comment:
2.8.1

Reply:
All ITs specify allure version '2.4.1' and adapter version '2.0-BETA21' I thought that it is some base versions that gradle plugin should support.
I think it is better to externalize this versions and keep consistent to get the benefit of caching.

Reply:
**All** ITs? Come on... :) If some people don't care about latest releases of [Allure](https://github.com/allure-framework/allure2/releases) and [Java adaptors](https://github.com/allure-framework/allure-java/releases), it doesn't really mean that everyone else should be confused with 1-year old versions. Moreover, there were lots of features / bugfixes introduced since 2.4.1 / 2.0-BETA21.

Reply:
Agree with latest release advantages, by the way this request offers fixes for latest gradle version, but by **all ITs** I mean gradle scripts for tests in [src/it](https://github.com/allure-framework/allure-gradle/tree/master/src/it) directory that specify [outdated allure versions](https://github.com/allure-framework/allure-gradle/blob/master/src/it/junit5/build.gradle#L17).

I'm afraid that update to the latest version in a single IT script makes no sense and lose caching benefits, while updating versions in all scripts is not a right thing for this request.